services/processor/native/repository/_Deployments/deploy.go

```diff
@@ -76,10 +81,29 @@ func _writeProcessor(serviceName string, processorType uint32) {
 	state.WriteUint32([]byte(serviceName+".Processor"), processorType)
 }
 
-func _readCode(serviceName string) []byte {
```

Comment:
In order to avoid a synched Prism and orbs-network-go upgrade, it's better to keep the API backward compatible even if it returns a wrong result (only part of the code).
Let's add a new function getCodePart(index uint32), where getCode is identical to getCodePart(0)

Reply:
From my side can merge after the fixes, verifying Prism is functional.

Reply:
I have verified that Prims did not break.